### ISSUE TITLE
Jetpack 5.0 - Issues report

### ISSUE THREAD

**Issue by leonpano2006** [rbonghi/jetson_stats#210]
Please update jetson-stats with new jetpack

**Linux for Tegra**
 - L4T: 34.1.0

**Jetson-Stats**
 - Version: 3.1.3

**Comment by leonpano2006** [comment]
and Fan is not working it will always show off even is runing on maxium speed

**Comment by rbonghi** [comment]
Hi @leonpano2006 
I will focus on JP5.0 soon. I think from next month.

Let me know all issues you find, and I will fix all.

**Comment by leonpano2006** [comment]
> Hi @leonpano2006 
> 
> I will focus on JP5.0 soon. I think from next month.
> 
> 
> 
> Let me know all issues you find, and I will fix all.

https://github.com/rbonghi/jetson_stats/issues/211
This is also one issue

**Comment by leonpano2006** [comment]
i also find other issue
memory only shows ram not vram
![image](https://user-images.githubusercontent.com/58368166/162568702-d84eec81-3647-475a-a7cf-eac42b235789.png)
unlike this one in jetpack 4.6.1
![image](https://user-images.githubusercontent.com/58368166/157514373-ed201a08-b656-4d6f-aab7-899e607d2fd8.png)
and meny info missing
![image](https://user-images.githubusercontent.com/58368166/162568751-7e017069-a605-4d9b-949a-5225588d136f.png)
on jetpack 4.6.1
![image](https://media.discordapp.net/attachments/588138053735022632/955392945794732082/AF2E0C8B-6F9C-44C4-A061-F371952E51F7.jpg)

**Comment by rbonghi** [comment]
Thank you!

**Comment by rbonghi** [comment]
Aligned and renamed this issue

**Comment by leonpano2006** [comment]
And FAN control function is failed

**Comment by leonpano2006** [comment]
lavapipe
similar to (that is jetpack 4.x with ubuntu 20.04) https://github.com/rbonghi/jetson_stats/issues/173
![image](https://user-images.githubusercontent.com/58368166/162591047-323f1f9a-aca8-478b-ae3b-2a9671a79698.png)

**Comment by phoenixmy** [comment]
I met same issue on Jetson Orin Dev Kit, maybe it is too new...


~$ jtop
[WARN] Board missing P3737-000 (press CTRL + Click)
[WARN] jetson-stats not supported for [L4T 34.1.0]
  Please, try: sudo -H pip install -U jetson-stats
  or open an issue on Github (press CTRL + Click)

<img width="1391" alt="image" src="https://user-images.githubusercontent.com/1450525/163682015-1eaf7b7c-71d8-4b69-b3d4-46d07a1c1bed.png">

**Comment by leonpano2006** [comment]
Of course 
Orin is not exist in l4t 32.x

**Comment by leonpano2006** [comment]
![image](https://user-images.githubusercontent.com/58368166/164393873-a89efdc6-e4b3-4ccf-a1f2-b965ade0aaaa.jpeg)
In jetpack 5.0 it can't detect ram use from gpu (only cpu)

**Comment by rbonghi** [comment]
I think is only missing to fix the GPU memory read and all major bugs for JP5 are fixed.
The new pre-release is the 4.0.0rc2
```
sudo -H pip install jetson-stats==4.0.0rc2
```

**Comment by dn1983** [comment]
Hi.
What about temperature?
![Screenshot from 2022-06-16 15-14-29](https://user-images.githubusercontent.com/33154165/174067847-a54258d4-afb1-445c-b14f-cb89f8a5e7b0.png)

**Comment by leonpano2006** [comment]
> Hi.
> 
> What about temperature?
> 
> ![Screenshot from 2022-06-16 15-14-29](https://user-images.githubusercontent.com/33154165/174067847-a54258d4-afb1-445c-b14f-cb89f8a5e7b0.png)
> 
> 

What is this absolute zero temperature

**Comment by rbonghi** [comment]
I think you installed a script that emulates fake data. Are you working on a desktop?

**Comment by rbonghi** [comment]
If you have an NVIDIA Jetson Orin, I cannot test, and I don't know this output.

**Comment by rbonghi** [comment]
This error should now be fixed with the latest release, **4.0** try to install and check again.
```
sudo -H pip install -U jetson-stats
```

The temperatures come directly from the Jetson sensors. So it is not an error from jtop. I hope that future Jetpack releases will fix them.
